troubleshooting/extra-android-permissions-always-set.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Extra Android permissions always set
+description: The APK built by the cloud service has more required permissions than I set in my project settings.

```

Comment:
I would go for something like this. Personal preference, so feel free to keep yours.
**The built application package (APK) requires more permissions than the specified in my project settings.**